server/datastore/psql/maintenance_requests.go

```diff
@@ -0,0 +1,35 @@
+package psql
+
+import (
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+// Import the necessary packages
+
+func (db *DB) AllMaintenanceRequests() ([]datastore.MaintenanceRequest, error) {
+	var maintenanceRequestTable []datastore.MaintenanceRequest
+	rows, err := db.connection.Query("SELECT * FROM maintenance_requests LIMIT $1", 777) //Number limits # of requests, can be anything
```

Comment:
wanted to ask is there a reason for the limit you put on the select statement?

Reply:
777 is a test roof limit, I can change it to a higher value, but I'm not sure if it can scan without the value